### PR TITLE
Radio Garden: update for new site design, new playing states

### PR DESCRIPTION
**Pull request by tepiloxtl**
Basically a rewrite of [radio.garden](http://radio.garden/) for new site design.

DiscordCanary_PunTYg7Csr
msedge_R7j9Wx9FK0
Playback stopped

DiscordCanary_n2sOKhXSOU
msedge_3ID5Tl58KE
Tuning into a station

msedge_v0a5SATkF0
DiscordCanary_g86pqnBhxv
Playing station, displaying station name and localisation

### CODE REVIEW

File: websites/R/Radio Garden/dist/metadata.json [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
+  "service": "Radio Garden",
   "description": {
     "en": "By bringing distant voices close, radio connects people and places. Radio Garden allows listeners to tune into live radio across entire globe."
   },
-  "service": "Radio Garden",
-  "version": "1.2.5",
+  "url": "radio.garden",
+  "version": "2.0.0",
   "logo": "https://vikavo.lt/premidpresence/radiogardenglobe.webp",
   "thumbnail": "https://vikavo.lt/premidpresence/radiogardenthumb.png",
```
Comment (ririxichan) [inline websites/R/Radio Garden/dist/metadata.json:14]: upload to imgur and use i.imgur links

File: websites/R/Radio Garden/presence.ts [hunk 2]
```diff
@@ -2,18 +2,48 @@ const presence = new Presence({
...
+let details: HTMLInputElement,
+state: HTMLInputElement;
 
-    presence.setActivity(presenceData);
-  }
-});
+presence.on("UpdateData", async () => {
+	if(!document.title.includes("Explore live radio by rotating")){
+		const elapsed =  Math.floor(Date.now() / 1000);
+		const presenceData: PresenceData = {
```
Comment (ririxichan) [inline websites/R/Radio Garden/presence.ts:11]: merge consts

File: websites/R/Radio Garden/presence.ts [hunk 3]
```diff
@@ -2,18 +2,48 @@ const presence = new Presence({
...
+//			details: `${document.querySelector(".ChannelTitle_title__2QQj5").innerText}`,
+//			state: `${document.querySelector(".ChannelTitle_subtitle__DZ_ZQ").innerText}`,
```
Comment (ririxichan) [inline websites/R/Radio Garden/presence.ts:13]: why commented?

File: websites/R/Radio Garden/presence.ts [hunk 4]
```diff
@@ -2,18 +2,48 @@ const presence = new Presence({
...
+		}
+//		This could work with some tweaks. It detects when youre not tuned into anything, but trips if radio youre tuned into is not in view...
+//		else {
+//			presenceData.smallImageKey = "statusstop";
+//			presenceData.smallImageText = "Tuning";
+//			presenceData.details = "Tuning";
+//			delete presenceData.state;
+//			delete presenceData.startTimestamp;
+//			console.log("broke")
+//		};
```
Comment (ririxichan) [inline websites/R/Radio Garden/presence.ts:47]: so delete this for now.

**Review by ririxichan — CHANGES_REQUESTED** [review]

File: websites/R/Radio Garden/dist/metadata.json (hunk 1 above)
Comment (tepiloxtl) [reply websites/R/Radio Garden/dist/metadata.json:14]: Changed

File: websites/R/Radio Garden/presence.ts (hunk 2 above)
Comment (tepiloxtl) [reply websites/R/Radio Garden/presence.ts:11]: Actually changed presenceData to let, for now it fixes some obscure behaviour until I figure out how to account for that.

File: websites/R/Radio Garden/presence.ts (hunk 3 above)
Comment (tepiloxtl) [reply websites/R/Radio Garden/presence.ts:13]: Updated and uncommented, also fixes the obscure behaviour from comment above

File: websites/R/Radio Garden/presence.ts (hunk 4 above)
Comment (tepiloxtl) [reply websites/R/Radio Garden/presence.ts:47]: Removed

File: websites/R/Radio Garden/presence.ts (hunk 2 above)
Comment (ririxichan) [reply websites/R/Radio Garden/presence.ts]: no, change it to const, it still should work. and merge consts

Comment (tepiloxtl) [reply websites/R/Radio Garden/presence.ts]: Changed, merged. It seems to break timer, for some reason? Ill make that work and push changes later

**Review by Bas950 — APPROVED** [review]

**Review by ririxichan — APPROVED** [review]

**Pull request merged**